test/ejabberd_tests/http.specs

```diff
@@ -0,0 +1,7 @@
+%% do not remove below SUITE if testing mongoose
+{suites, "tests", mod_http_notification_SUITE}.
```

Comment:
Please add the `mod_http_notification_SUITE` to `default.spec` so travis can test it.